addon/components/ember-popper-base.js

```diff
@@ -27,6 +27,12 @@ export default class EmberPopperBase extends Component {
   @property modifiers = null
 
   /**
+   * An optional function to be called when a new target is located.
+   * The target is passed in as an argument to the function.
+   */
+  @property onFoundTarget = function() {}
```

Comment:
This is probably not ideal, creates an extra closure. I would set it to `null` if it's expected to be a function.

Reply:
fair enough. Was debating between what I have now and null, the disadvantage being checking for the existence of the action.